MyApplication/app/src/main/AndroidManifest.xml

```diff
@@ -17,6 +17,16 @@
             </intent-filter>
         </activity>
 
+        <service android:name=".services.MyService"></service>
+
+        <!--<receiver android:name=".activities.MainActivity$MyReceiver">-->
```

Comment:
Ai comentat partea asta de receiver ca nu-l mai folosim, da? Ca parca asa imi aduc aminte. Ii ok si asta, dar asa ca si sfat in general pe partea de development: din momentul in care nu mai este nevoie de anumite linii de cod, acestea se scot complet. Daca e nevoie sa fie comentate datorita unei anumite motive, ca pe viitor sa fie decomentat, e nevoie si de un comentariu pentru a motiva punerea liniilor de cod sub forma de comments. :)